src/OrderFormAcceptanceTests.TestData/SupplierInfo.cs

```diff
@@ -30,6 +30,19 @@ FROM CatalogueItem
             return await SqlExecutor.ExecuteAsync<SupplierDetails>(connectionString, query, new { catalogueItemType = (int)catalogueItemType });
         }
 
+        public static async Task<string> GetSupplierWithMultipleAssociatedServices(string connectionString)
+        {
+            var query = @"SELECT TOP (1000) s.Id
+                          FROM dbo.AssociatedService AS a
+                          INNER JOIN CatalogueItem ci ON ci.CatalogueItemId = a.AssociatedServiceId
+                          INNER JOIN Supplier s on s.Id = ci.SupplierId
```

Comment:
Recommend using schema name, `dbo`, for all tables and AS for all table aliases. The last `ON` should be upper case as well primarily for consistency. 😃 

```sql
INNER JOIN dbo.CatalogueItem AS ci -- ...
INNER JOIN dbo.Supplier AS s ON -- ...
```

Reply:
I'll do a bigger update when fewer stories are in motion to add `dbo.` to tables